websites/O/OpenStreetMap/presence.ts

```diff
@@ -0,0 +1,201 @@
+const presence = new Presence({
+		clientId: "983277228710297600"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let locationcity: string,
+	locationstate: string,
+	locationcountry: string,
+	x: string,
+	y: string,
+	editedObject: string,
+	selectedObject: boolean;
+setInterval(() => {
+	if (x && y) {
+		fetch(
+			`https://nominatim.openstreetmap.org/reverse?lat=${x}&lon=${y}&format=json`
+		)
+			.then(res => res.json())
+			.then(res => {
+				locationcity = res.address.city;
+				if (!locationcity) locationcity = res.address.hamlet;
+				if (!locationcity) locationcity = res.address.village;
+				if (!locationcity) locationcity = res.address.town;
+
+				locationstate = res.address.state;
+				locationcountry = res.address.country;
+			})
+			.catch();
+	}
+}, 5000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
+		startTimestamp: browsingTimestamp,
+		details: "Current page:"
```

Comment:
you shouldn't do this. Just display each page's data in the details, maybe change the strings a little to say "Viewing" or whatever you'd like but don't do this